src/exec/Makefile

```diff
@@ -19,7 +19,7 @@ ifeq ($(OCCLUM_RELEASE_BUILD), 1)
 	@RUSTC_BOOTSTRAP=1 RUSTC_WRAPPER=$(RUSTC_WRAPPER) cargo build --release --target-dir=$(RUST_TARGET_DIR) -Z unstable-options --out-dir=$(RUST_OUT_DIR)
 	@echo "CARGO (release) => exec"
 else
-	@RUSTC_BOOTSTRAP=1 RUSTC_WRAPPER=$(RUSTC_WRAPPER) cargo build --target-dir=$(RUST_TARGET_DIR) -Z unstable-options --out-dir=$(RUST_OUT_DIR)
+	@RUSTC_BOOTSTRAP=1 RUSTC_WRAPPER=$(RUSTC_WRAPPER) RUSTFLAGS=-Zsanitizer=address RUSTDOCFLAGS=-Zsanitizer=address cargo build --target-dir=$(RUST_TARGET_DIR) -Z unstable-options --out-dir=$(RUST_OUT_DIR) -Zbuild-std --target x86_64-unknown-linux-gnu
```

Comment:
I am not sure that we always want to enable sanitizers (e.g., address sanitizer) even for debug builds. According to my past experience, enabling sanitizers require code instrumentation or other compiler black magics, which means non-trivial runtime cost. Furthermore, the benefits of these sanitizers is minimal since 1) Rust is mostly free from memory safety bugs and 2) the chances of memory leakage problems (except for the trivial ones fixed in this PR) is not high given the fact that the complexity of occlum exec client/server is quite limited. Due to this cost-benefit analysis, **my best judgement is to enable sanitizers only on demand, not by default.**